modules/ROOT/pages/create-api-specification-studio.adoc

```diff
@@ -0,0 +1,36 @@
+= Create an API Project in Studio
+
+Anypoint Studio 7.7 and later enables you to create a RAML or OAS 2.0 and 3.0 API specification, or an API fragment in Studio and automatically synchronize it with Design Center.
```

Comment:
I would add a comment related to the limitation of OAS3.0 in mule projects. See https://github.com/mulesoft/Mule-Tooling/pull/2846